Example/Podfile

```diff
@@ -4,3 +4,7 @@ use_frameworks!
 target 'ReusableDemo' do
   pod 'Reusable', :path => '../'
 end
+
+post_install do |installer|
+  system "sed -i '' -E 's/EMBEDDED_CONTENT_CONTAINS_SWIFT[[:space:]]=[[:space:]]YES/EMBEDDED_CONTENT_CONTAINS_SWIFT = NO/g' Pods/Target\\ Support\\ Files/Pods-ReusableDemo/*.xcconfig"
```

Comment:
That `EMBEDDED_CONTENT_CONTAINS_SWIFT` issue is very strange though. No sure why we need to disable it. Or maybe we shouldn't, and we should simply add `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIEES=YES` to the mix (until CocoaPods does it itself)?